t/RPC/exceptions.t

```diff
@@ -0,0 +1,82 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Deep;
+use Test::Fatal;
+use Test::Myriad;
```

Comment:
We shouldn't need to do any of this - we have `use Myriad::Class` for the standard Perl features, and `use Test::Myriad` for test-specific details, better to have that pull in any test-related components (can be addressed in a separate branch though).

Reply:
I have tried with `Myriad::Class` but it produce this error
 ```
Prototype mismatch: sub main::blessed ($) vs none at t/RPC/exceptions.t line 3.
```